projects/plugins/jetpack/modules/theme-tools/social-menu/icon-functions.php

```diff
@@ -98,8 +98,24 @@ function jetpack_social_menu_nav_menu_social_icons( $item_output, $item, $depth,
 		// Change SVG icon inside social links menu if there is supported URL.
 		if ( 'jetpack-social-menu' === $args->theme_location ) {
 			foreach ( $social_icons as $attr => $value ) {
-				if ( false !== strpos( $item_output, $attr ) ) {
-					$item_output = str_replace( $args->link_after, '</span>' . jetpack_social_menu_get_svg( array( 'icon' => esc_attr( $value ) ) ), $item_output );
+				/*
+				 * attr can be a URL host, or a regex, starting with #.
+				 * Let's check for both scenarios.
+				 */
+				if (
+					// First Regex.
+					(
+						false !== stripos( $attr, '#' )
```

Comment:
We could check for both the first and the last symbol to be `'#'`, to decrease the number of potential edge cases:
```suggestion
						'#' === substr( $attr, 0 ) && '#' === substr( $attr, -1 )
```

Do you think that would make sense?

Reply:
I like the idea. I did this in 2747e71c689c44cfd3bd54881b0731308f03c31c